tests/suse_migration_services.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test suse_migration_services
+#
+# Maintainer: QE C <qe-c@suse.de>
+#
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration qw(runtime_registration deregister_addons_cmd);
+use version_utils qw(get_os_release);
+use Config::Tiny;
+use Test::Assert ':all';
+use power_action_utils 'power_action';
+use serial_terminal 'select_serial_terminal';
+
+
+my $suse_migration_services_repo = get_var("SUSE_MIGRATION_SERVICES_REPO", "https://download.opensuse.org/repositories/home:/marcus.schaefer:/dms/SLE_12_SP5");
+my $upgrade_expected_version = get_var("UPGRADE_EXPECTED_VERSION", "15-SP5");
+
+sub print_debug_info {
+    select_serial_terminal;
+
+    record_info("cat /etc/os-release", script_output("cat /etc/os-release", proceed_on_failure => 1));
+    record_info("python3 --version", script_output("python3 --version", proceed_on_failure => 1));
+    record_info("cat /usr/bin/suse-migration-pre-checks", script_output("cat /usr/bin/suse-migration-pre-checks", proceed_on_failure => 1));
```

Comment:
```suggestion
    record_info("DEBUG", script_output("cat /usr/bin/suse-migration-pre-checks", proceed_on_failure => 1));
```

please check your VRs , first parameter is what to be displayed in small square and everything longer then 5-8 chars will be cut anyway . so best what we can do is follow "one word limit" . "DEBUG" is just suggestion feel free to replace it with any other word but your version certainly needs to be replaced